spec/factories/projects.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :project do
+    name { Faker::Name.name }
```

Comment:
add `association :user`

Reply:
or simply 
```
factory :project do
  name { Faker::Name.name }

  user # < this will build association

  # or use #association method to build customized association
  association :user, :with_some_trait
end
```